ex1.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
```

Comment:
Jeśli ten skrpyt wykonuje jakieś konkretne zadanie , w tym przypadku (Skrypt wykonujący 'IsTheSame') nazwa pliku powinna to odwzierciedlać. Usunąć komentarze, zmienić nazwe pliku

Reply:
Dotyczy to wszystkich plików